tools/udisksctl.c

```diff
@@ -1564,24 +1571,15 @@ handle_command_loop (gint        *argc,
 
   if (request_completion)
     {
-      if (is_setup)
+      if (!complete_files && !complete_devices && !complete_objects)
         {
-          if (opt_loop_file == NULL && !complete_files)
-            {
-              g_print ("--file \n");
-            }
-          if (complete_files)
-            {
-              g_print ("@FILES@");
-            }
-        }
+	  list_options (is_setup ? command_loop_setup_entries : command_loop_delete_entries);
```

Comment:
Please only use spaces for indentation.

Reply:
Tabs, missing space and extracting the variable from the for loop are now also included in the fresh push.